horovod/torch/__init__.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, params, named_parameters, compression,
             self._parameter_names = {v: 'allreduce.noname.%s' % i
                                      for param_group in self.param_groups
                                      for i, v in enumerate(param_group['params'])}
+            # named_parameters = [(k, v) for v, k in self._parameter_names.items()]
```

Comment:
Thanks! Should this be uncommented?

Reply:
Yes. I was testing that my tests failed properly on BuildKite before uncommenting. In the process, I found that I had an error in my test. I will finish soon.